test/__snapshots__/integration.test.ts.snap

```diff
@@ -141,7 +120,7 @@ Object {
     "content": Object {
       "mediaEncoding": Object {
         "__typename": "UnsupportedEncodingTypes",
-        "original": "https://api.zora.co/media/ETHEREUM-MAINNET/0xca21d4228cdcc68d4e23807e5e370c07577dd152/4/content/original",
+        "original": "https://api.notzora.com/media/ETHEREUM-MAINNET/0xca21d4228cdcc68d4e23807e5e370c07577dd152/4/content/original",
```

Comment:
this url will need to be updated after the test target is changed

Reply:
good catch!